cmd/nomos/version/version.go

```diff
@@ -96,12 +96,11 @@ of the "nomos" client binary for debugging purposes.`,
 func allKubectlConfigs() (map[string]*rest.Config, error) {
 	allCfgs, err := restconfig.AllKubectlConfigs(flags.ClientTimeout)
 	if err != nil {
-		// Unwrap the "no such file or directory" error for better readability
-		if unWrapped := errors.Cause(err); os.IsNotExist(unWrapped) {
-			err = unWrapped
+		var pathErr *os.PathError
+		if errors.As(err, &pathErr) {
+			err = pathErr
 		}
 
-		// nolint:errcheck
```

Comment:
Does this need to be removed? Looks unrelated to this change.

Reply:
removed.